src/main/java/com/pinterest/secor/parser/MessageParser.java

```diff
@@ -50,6 +52,10 @@ public MessageParser(SecorConfig config) {
         }
     }
 
+    static String usingOffsetPrefix(SecorConfig config) {
+        return config.getString("secor.offsets.prefix", "offset=");
```

Comment:
In secor, we keep the convention that the default value is specified in secor.common.properties, not in Java code.  You don't need the 2nd argument for default